modules/apps/adaptive-media/adaptive-media-blogs-web/src/main/java/com/liferay/adaptive/media/blogs/web/internal/optimizer/BlogsAMImageOptimizer.java

```diff
@@ -137,7 +149,7 @@ private void _optimize(
 		}
 	}
 
-	private void _sendStatusMessage(int count, int total) {
+	private void _sendStatusMessage(int count, int total, int errors) {
```

Comment:
Can we sort the parameters so that it looks like `_sendStatusMessage(int count, int errors, int total)`?